packages/transformers/posthtml/src/PostHTMLTransformer.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+
+import posthtml from 'posthtml';
+import parse from 'posthtml-parser';
+import render from 'posthtml-render';
+import nullthrows from 'nullthrows';
+import semver from 'semver';
+
+const getPostHTMLConfig = async asset => {
+  let config = await asset.getConfig(
+    ['.posthtmlrc', '.posthtmlrc.js', 'posthtml.config.js'],
+    {
+      packageKey: 'posthtml'
+    }
+  );
+  if (!config && !asset.options.minify) {
```

Comment:
We should move minifying into an optimizer (this can definitely be another PR), so can just remove these lines here.

An aside: is `options` a thing on asset? I think we should use the parcel `options` passed into all the transformer methods. But again minify stuff should be in an optimizer.

Reply:
I will remove, good call. This was directly ported and I must have glazed over this part.